src/view/printer/warn.ts

```diff
@@ -1,4 +1,4 @@
-import * as chalkDefault from 'chalk';
+const chalkDefault = require('chalk');
```

Comment:
require() is legacy way to import something using CommonJS. Use the ESM way for future proofing: https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

Reply:
rule disabled